backend/corpora/common/corpora_orm.py

```diff
@@ -183,32 +183,32 @@ class DbProject(Base):
     data_submission_policy_version = Column(String, nullable=False)
 
     # Relationships
-    links = relationship("DbProjectLink", back_populates="project", cascade="all, delete-orphan")
-    datasets = relationship("DbDataset", back_populates="project", cascade="all, delete-orphan")
+    links = relationship("DbCollectionLink", back_populates="collection", cascade="all, delete-orphan")
```

Comment:
this will require changing the table name to collections which requires a migration